src/geoserverx/cli/cli.py

```diff
@@ -38,51 +43,102 @@ def workspaces(
     ),
     password: str = typer.Option("geoserver", help="Geoserver Password"),
     username: str = typer.Option("admin", help="Geoserver username"),
+    output: str = typer.Option("table", help="Output format - 'json' or 'table'"),
 ):
     """
     Get all workspaces in the Geoserver
+    looks like - gsx workspaces --url <url> --username <username> --password <password>
     """
     if request.value == "sync":
         client = SyncGeoServerX(username, password, url)
-        result = client.get_all_workspaces().model_dump_json()
+        result = client.get_all_workspaces()
         if "code" in result:
             typer.secho(result, fg=typer.colors.RED)
         else:
-            print(result)
+            if output == "json":
+                print(json.dumps(result.model_dump(), indent=2))
```

Comment:
pydantic's [model_dump_json()](https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_dump_json) is able to indent: 

```suggestion
                print(result.model_dump_json(indent=2))
```

Reply:
This comment applies to all places where this new output parameter is used, not just in this specific place - be sure to change them all